src/main.c

```diff
@@ -78,7 +78,7 @@ struct HBA *parsed_hba;
 
 int cf_daemon;
 int cf_pause_mode = P_NONE;
-int cf_shutdown; /* 1 - wait for queries to finish, 2 - shutdown immediately */
+int cf_shutdown;/* 1 - wait for queries to finish, 2 - shutdown immediately */
```

Comment:
strange

Reply:
Well spotted. This was a weird thing. uncrustify is trying to align the comment on a tabstop turns out is right after the `;`.  I added a tab manually to give it some spacing. For some reason it wouldn't do it automatically. I think in a follow up PR I might propose changing to `indent_cmt_with_tabs = false`, which did add a single space between the semicolon and the comment. But for this PR that would definitely change too many places.

Reply:
ok

Reply:
seems ok after your change